# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vdr/dvd-gccoptions.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- dvd-old/Makefile.orig	2004-06-28 17:34:47.000000000 +0200
+++ dvd/Makefile	2004-06-28 17:42:44.644468916 +0200
@@ -16,7 +16,7 @@
 ### The C++ compiler and options:
 
 CXX      ?= g++
-CXXFLAGS ?= -O3 -Wall -Woverloaded-virtual
+CXXFLAGS ?= -O2 -Wall -Woverloaded-virtual
 LDFLAGS  ?= $(CXXFLAGS)
 
 ### The directory environment:
@@ -57,13 +57,13 @@
 LIBS   += -L/usr/lib/djbfft/lib -ldjbfft
 endif
 
-ifdef DBG
-CXXFLAGS += -g -ggdb -O0
-LDFLAGS  += -g -ggdb -O0
-else
-CXXFLAGS += -O3
-LDFLAGS  += -O3 -Wl,--retain-symbols-file,retain-sym
-endif
+#ifdef DBG
+#CXXFLAGS += -g -ggdb -O0
+#LDFLAGS  += -g -ggdb -O0
+#else
+#CXXFLAGS += -O3
+#LDFLAGS  += -O3 -Wl,--retain-symbols-file,retain-sym
+#endif
 
 ### The object files (add further files here):
 
